google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -227,8 +230,11 @@ public CloudSdk createSdk(
         .appCommandCredentialFile(credentialsPath.toFile())
         .appCommandMetricsEnvironment(pluginInfoService.getExternalPluginName())
         .appCommandMetricsEnvironmentVersion(pluginInfoService.getPluginVersion())
-        .appCommandOutputFormat("json")
-        .build();
+        .appCommandOutputFormat("json");
+
+    getProjectJavaSdk(project).ifPresent(sdkBuilder::javaSdkPath);
```

Comment:
Because the Cloud SDK might be created for running processes that don't use Java, I think it's better to allow passing here without a JDK.
We're failing in other places before creating the Cloud SDK if there is no JDK set. Like before running the local server.